### DISCUSSION TITLE
Autopilot Tool Upload Error?

### DISCUSSION THREAD

**Discussion by Stmaki1997** [Jaekty/Autopilot-Management#3] [Q&A]
CSV Upload with list of Serial Numbers never finds the machines. Keeps spitting out the text file of machines it can't find. If you put each one in individually it finds them. Confirmed not held back by cache, proper header name, csv file. Did notice there was a bugfix for 1.1.1

Downloaded 1.1.1 today so would assume that shouldn't be the issue.

Any help would be appreciated.

Thanks!

**Comment by Jaekty** [comment]
Are you attempting to "Upload Hash" (button) or are you loading csv to "Query" existing devices?
Tested some csv queries and it's working as it should on my side. Same with upload.

If you're querying, can you try creating a new txt/csv with only header "Device Serial Number" then insert all the serial numbers you want to query. F.ex. like this:
Device Serial Number
1234-123-111
4321-321-222
1111-321-123

Only the "Device Serial Number"-header within the csv when querying.

**Reply by Stmaki1997** [reply]
I am loading a csv to query existing devices. I have the header as Device Serial Number and I only put 2 serial numbers. No other headers. 
![Screenshot 2023-09-26 071618](https://github.com/Jaekty/Autopilot-Management/assets/129766888/57ef09ea-cd35-4fe3-81d5-d3ec15ab1a2c)

**Reply by Jaekty** [reply]
Can you try running the PowerShell-script through PowerShell ISE to see if any errors appear?
Can you also confirm you do not have any space after the serial number (I should probably implement a trim).
